bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,187 @@
+# Babylon Validator Setup
+
+## Table of Contents
+
+1. [Prerequisites](#prerequisites)
+2. [Setup](#setup) \
+   2.1. [Setup Node and Configure](#setup-node-and-configure) \
+   2.2. [Build node from `x` tag](#build-node-from-x-tag) \
+   2.3. [Initialize the chain](#initialize-the-chain) \
+   2.4. [Update the configuration](#update-the-configuration) \
+   2.5. [Retrieve the genesis file](#retrieve-the-genesis-file) \
+   2.6. [Create a Keyring](#create-a-keyring) \
+   2.7. [Get Funds](#get-funds) \
+   2.8. [What is BLS Voting](#what-is-bls-voting) \
+   2.9. [Create BLS Key](#create-bls-key)
+3. [Run Node](#run-node)
+4. [Create Validator](#create-validator)
+5. [Verify your Validator](#verify-your-validator)
+
+## Prerequisites
+
+This system spec has been tested by validators and found to be comfortable:
+
+- Quad Core or larger AMD or Intel (amd64) CPU
+- 32GB RAM
+- 1TB NVMe Storage
+- 100MBps bidirectional internet connection
+
+You can run Babylon on lower-spec hardware for each component, but you may find that it is not highly performant or prone to crashing.
```

Comment:
```suggestion

Note that the above is a sample system specification that might not fit your specific infrastructure.
Please do your own research before committing to a setup and verifying that it can scale well
depending on your operations and needs.

```